src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -1080,6 +1081,7 @@ def aks_create(cmd,     # pylint: disable=too-many-locals,too-many-statements,to
         proximity_placement_group_id=ppg,
         availability_zones=node_zones,
         enable_node_public_ip=enable_node_public_ip,
+        enable_fips=enable_fips_image,
```

Comment:
This change in this file and file blow covers both command 'az aks create...' and 'az aks nodepool add'?

Reply:
there is another enable_fips=enable_fips_image in this file to cover nodepool add.